src/assets/text/modelingText.js

```diff
@@ -1,33 +1,33 @@
                 'flubber_id': 'ANN1',
                 'bees_id': 'step_error'
             },
             {
-                'method': "The first component of an ANN is called an input layer – this is where the model receives the raw input data.",
+                'method': "The first component of an ANN is called an input layer – this is where the model receives the input data.",
                 'flubber_id': 'ANN2',
                 'bees_id': 'step_error_exp'
             },
             {
-                'method': "From the input layer, the information is passed to what is known as a ‘hidden’ layer through a series of connections. Each connection transmits an input value to a processing node in the hidden layer. The model can vary the weight of these connections to assign more or less importance to a particular piece of information.",
+                'method': "From the input layer, the information is passed to what is known as a ‘hidden’ layer through a series of connections. Each connection transmits an input value to a node in the hidden layer. During training, the model can vary the weight of each connection to assign more or less importance to a particular piece of information.",
                 'flubber_id': 'ANN3',
                 'bees_id': 'step_error_exp'
             },
             {
-                'method': "In the hidden layer, the receiving nodes compute a weighted sum of the received information…",
+                'method': "In the hidden layer, the receiving nodes calculate a weighted sum of the received information…",
                 'flubber_id': 'ANN4',
                 'bees_id': 'step_error_exp'
             },
             {
-                'method': "…and then pass the weighted sum through a mathematical function, producing a single output value for each node. ",
+                'method': "…and then the weighted sum is adjusted by some simple rule, for example, keeping positive values as-is and setting negative values to zero.",
                 'flubber_id': 'ANN5',
                 'bees_id': 'step_error_exp'
             },
             {
-                'method': "These output values are then passed to nodes in the next hidden layer, or, if there are no more hidden layers, to the output layer.",
+                'method': "These output values are then passed to nodes in the next hidden layer, or, if there are no more hidden layers, to the output layer. Adding nodes or layers to the model increases the complexity of relationships between inputs and outputs the model is able to mimic.",
```

Comment:
I like this addition here about more layers = capture more complex relationships